apps/web/styles/emoji.css

```diff
@@ -15,8 +17,8 @@
 
 .epr-emoji-category-label {
   font-size: 0.7875rem !important;
-  color: rgba(var(--color-text-300)) !important;
   background-color: rgba(var(--color-background-100), 0.8) !important;
+  @apply !text-tertiary;
```

Comment:
v4 puts ! at the end ! at the front is for compatibility of v3.

https://github.com/tailwindlabs/tailwindcss/discussions/15803

Reply:
all of this should go in the `@components` layer too, once you've figured out the layers stuff you might not even need `!` because specificity will be higher (correct)